snare/tests/test_add_scheme.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import sys
+import os
+import yarl
+import shutil
+from snare.cloner import Cloner
+from snare.utils.page_path_generator import generate_unique_path
+
+
+class TestCloner(unittest.TestCase):
+    def setUp(self):
+        self.url = 'http://example.com/'
+        self.main_page_path = generate_unique_path()
+        os.makedirs(self.main_page_path)
+        self.expected_new_url = yarl.URL('http://example.com')
+        self.expected_err_url = yarl.URL('http://example.com/status_404')
+        self.max_depth = sys.maxsize
+        self.css_validate = 'false'
+        self.handler = Cloner(self.url, self.max_depth, self.css_validate)
+
+    def test_add_scheme(self):
```

Comment:
Add a test case for [this](https://github.com/mushorg/snare/blob/master/snare/cloner.py#L34)